lib/assessment.md

```diff
@@ -22,14 +22,23 @@ Using this format will make it easier for us to grade you, so that we can get ev
 
 When we say that the web is a service, what does that mean? Explain the interaction in terms of clients and servers.
 <!-- your answer starts here -->
-
+The web is a service that uses the internet. A service has clients and servers. Someone who asks and someone who responds.
 <!-- your answer ends here -->
 
 ## Question 2
 
 Between the comments below, write HTML to create a basic web page (using the format described in the instructions). The page should have a title in the head and an unordered list (with three items) inside the body.
 <!-- your answer starts here -->
-
+<html>
+  <head>The Title of Webpage X</head>
+    <body>
+      <ul>
+      <li>Here are</li>
+      <li>somethings</li>
+      <li>on a list</li>
+      </ul>
+    </body>
+  </html?
```

Comment:
Code is correct (minus the typo in the final line). However, your indenting is not correct. `<head>` and `<body>` should be equally indented since they are sibling elements. `<li>` should be indented from `<ul>` because it is a child element of `<ul>`.